mailpoet/tests/performance/tests/newsletter-searching.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-default-export */
+/**
+ * External dependencies
+ */
+import { sleep, check, group } from 'k6';
+import { chromium } from 'k6/experimental/browser';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+  baseURL,
+  thinkTimeMin,
+  thinkTimeMax,
+  headlessSet,
+  timeoutSet,
+} from '../config.js';
+import { authenticate } from '../utils/helpers.js';
+/* global Promise */
+
+export function newsletterSearching() {
+  const browser = chromium.launch({
+    headless: headlessSet,
+    timeout: timeoutSet,
+  });
+  const page = browser.newPage();
```

Comment:
```
  const browser = chromium.launch({
    headless: headlessSet,
    timeout: timeoutSet,
  });
  const page = browser.newPage();
```
One suggestion would be to extract this to a helper function that returns the browser and page, I noticed it is the same for all tests.
Maybe include also the authentication.

Reply:
I see this as a widely accepted behavior to leave in every test and not in a helper.js, I don't feel necessary to be in the helper. If you have some example how it would be called in the helper, please let me know.